options/logger.go

```diff
@@ -132,6 +133,19 @@ type loggerConfigInfo struct {
 	Config RawLoggerConfig       `json:"config" bson:"config"`
 }
 
+// NewLoggerConfig returns a LoggerConfig with the given info, this function
+// expects a raw logger config as a byte slice. When using a LoggerProducer
+// directly, use LoggerConfig's Set function.
+func NewLoggerConfig(producerType string, format RawLoggerConfigFormat, config []byte) *LoggerConfig {
```

Comment:
can you pass a LoggerProducer rather than it as JSON?

Reply:
We have Set for that. This is a special case when you have the raw log config but potentially not the logger producer (i.e. buildlogger v3's setup being in timber)